crates/metrics/tests/proc_macro_attr.rs

```diff
@@ -0,0 +1,16 @@
+//! This test aims to check the usage of the latency attribute proc macro with the holochain's metric crate.
+
+#[cfg(test)]
+mod tests {
+    use holochain_metrics::prelude::*;
+
+    #[latency]
+    pub fn test_latency() {
+        ::std::thread::sleep(::std::time::Duration::from_millis(1001));
+    }
+
+    #[test]
+    fn main_test() {
+        eprintln!("Ahoy !");
+    }
```

Comment:
Is there any way to make this into an actual test? Or is the metric publisher not configurable to specify where things get published to? @struktured